src/services/types.ts

```diff
@@ -333,6 +333,7 @@ namespace ts {
 
     export interface GetCompletionsAtPositionOptions {
         includeExternalModuleExports: boolean;
+        includeBracketCompletions: boolean;

```

Comment:
If we ever want to support completing strings in object literals (without users needing to explicitly type in a quote character), does it make sense to call this something else?

Reply:
It depends on whether we will still want a flag for the *behavior* of completing bracketed expressions, or if this will only be used in a non-user-facing way as an editor feature flag. I believe vscode had plans to make a user-facing options for `includeExternalModuleExports` in case some people don't want that.

Reply:
Yes we have an option for that `includeExternalModuleExports`. However I think we would always want to enable `includeBracketCompletions` or whatever this new option is called. This flag is useful for backwards compatibility (see #20730) so that we don't break clients that don't support `insertText`